lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.pop(-2)
 
```

Comment:
Close! we would pass `2` as an argument not `-2`
Pop takes a parameter in Ruby of how many elements we want to remove